packages/ckeditor5-table/src/ui/colorinputview.js

```diff
@@ -208,6 +208,7 @@ export default class ColorInputView extends View {
 
 		dropdown.buttonView.children.add( colorPreview );
 		dropdown.buttonView.tooltip = t( 'Color picker' );
+		dropdown.buttonView.label = t( 'Color picker' );
```

Comment:
Now the label and tooltip are duplicated. These should remain synchronized.

I believe that setting `tooltip` to `true` will cause it to use the label, at least [that's what the docs are saying](https://ckeditor.com/docs/ckeditor5/latest/api/module_ui_button_button-Button.html#member-tooltip).

> If defined as a `Boolean` (e.g. `true`), then combination of `label` and `keystroke` will be set as a tooltip.